app/date-comment-helpers.js

```diff
@@ -34,50 +34,43 @@ class DateCommentHelpers {
     if (correctComment) {
       const commentDateMatch = myRegex.exec(correctComment.data.text);
       const commentDate = commentDateMatch[1];
-      const commMoment = moment(commentDate, 'MM/DD/YYYY').toISOString();
-      return commMoment;
+      const isoDate = moment(commentDate, 'MM/DD/YYYY').toISOString();
+      return isoDate;
     }
     return false;
   }
 
-  findPrevMoveDateFromComments(opts) {
-    const actionList = opts.actionList;
-    const cardCreationDate = opts.cardCreationDate;
-    const commentList = opts.commentList;
-    let correctComment = false;
-    if (commentList) {
-      correctComment = this.checkCommentsForDates(commentList, true, false);
-    }
-    if (correctComment) {
-      return correctComment;
-    } else if (actionList) {
-      if (actionList.length > 1) {
-        return actionList[0].date;
-      }
+  getNewCommentFromDate(deletedNewComment, comments, altToDate) {
+    if (altToDate) {
+      return moment(this.checkCommentsForDates(comments, true, true));
     }
+    const useLastFromDate = !deletedNewComment;
+    return moment(this.checkCommentsForDates(comments, true, useLastFromDate));
```

Comment:
Could this just be `return moment(this.checkCommentsForDates(comments, true, !deletedNewComment);` and then not need the `const` declaration?  Though doing it this way adds some context for what that last argument is, so :shrug: I guess.  😛